routers/web/repo/view.go

```diff
@@ -842,6 +842,7 @@ func renderDirectoryFiles(ctx *context.Context, timeout time.Duration) git.Entri
 	// Show latest commit info of repository in table header,
 	// or of directory if not in root directory.
 	ctx.Data["LatestCommit"] = latestCommit
+	var latestCommitID string
 	if latestCommit != nil {
```

Comment:
Looks like `entries.GetCommitsInfo`, it makes sure `lastestCommit` is not nil if err is nil. So I think we can remove this line.